### PR TITLE
Full Gids in URLs. Also, disable magnitudes when loading.

### PR DESCRIPTION
**Pull request by rpiaggio**
We now use the canonical form of `Gid`s in URLs. This creates consistency with the API and everywhere else. Also, it seems to resolve [ch-534] by sidestepping the issue with Safari.

Also, we now disable magnitude editing whenever the rest of of the target editor fields are disabled.

### CODE REVIEW

**Review by cquiroz — APPROVED** [review]
Looks Good, do you know if we can disable `verify` in production?

**Comment by rpiaggio** [comment]
> Looks Good, do you know if we can disable `verify` in production?

That's a good idea, let me give it a try.

File: targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala [hunk 1]
```diff
@@ -124,15 +126,17 @@ object MagnitudeForm {
...
                           )
                         )
                       ),
                       TableCell(
                         <.span(
                           EnumViewSelect(
                             id = magnitude.band.toString + "_SYSTEM",
-                            value = magnitudeView.zoom(Magnitude.system)
+                            value = magnitudeView.zoom(Magnitude.system),
+                            disabled = props.disabled
```
Comment (toddburnside) [inline targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala:139]: Should all the buttons on the form be disabled, too?

Comment (rpiaggio) [reply targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala:139]: Yes, that's a good idea.

**Pull request merged**